### DISCUSSION TITLE
Add a option in the status page that you can select if you want the dark/light theme

### DISCUSSION THREAD

**Discussion by MatadorProBr** [upptime/upptime#429] [Ideas] [answered]
The white theme is not very good for most people as it is very bright, but also the dark theme is not very good for some people and also the black theme is not very good for some people as it is too black (or something like that), so one thing that would be really nice to add is an option that lets you choose whether you want to use the black or light theme.

Something like that button:
![image](https://user-images.githubusercontent.com/71022656/129212222-bce05f76-010d-4704-85d9-25b60b9c4b31.png)
![image](https://user-images.githubusercontent.com/71022656/129212182-2a9963a9-1885-4a0c-8b32-e9874441a62c.png)

**Comment by titanism** [comment, marked as answer]
https://github.com/upptime/status-page/pull/517

**Reply by AnandChowdhary** [reply]
This should be landing soon!
